tinygrad/renderer/__init__.py

```diff
@@ -113,6 +113,10 @@ def launch_dims(self, var_vals:dict[Variable, int]):
     local_size = [sym_infer(sz, var_vals) for sz in self.local_size] if self.local_size is not None else None
     return global_size, local_size
 
+def bf16_bitcast_to_float(buf:UOp, idx:UOp, root:UOp):
```

Comment:
Hmm, I don't think this belongs in init. The right place is probably in rewriter.py, see `get_late_rewrite_patterns`